assets/chef-recipes/cookbooks/mariadb/recipes/mdberepos.rb

```diff
@@ -25,6 +25,17 @@
         sensitive true
       end
     end
+    if node['mariadb']['repo'].include?('es-repo.mariadb.net')
+      remote_file "/tmp/#{repo_file_name}.public" do
+        source node['mariadb']['repo_key']
+      end
+      execute 'install key' do
+        command "apt-key add /tmp/#{repo_file_name}.public"
```

Comment:
Usage of apt-key is deprecated. Use repository keys instead: https://askubuntu.com/questions/1286545/what-commands-exactly-should-replace-the-deprecated-apt-key